src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -139,79 +139,79 @@
     <value>Background</value>
     <comment>This is the header for a control that lets the user select the background color for text displayed on the screen.</comment>
   </data>
-  <data name="ColorScheme_Black.Header" xml:space="preserve">
+  <data name="ColorScheme_Black.Text" xml:space="preserve">

```

Comment:
This will force a relocalization of all the color strings -- just FYI -- which may impact the stable build loc. I'll file a followup on myself to check.

Reply:
thank you!